dashboard/app/views/projects/projects.html.haml

```diff
@@ -15,8 +15,31 @@
       font-size: 1.5em;
     }
   }
-%div{'ng-controller' => 'ProjectsController'}
-  
+
+  .level-buttons {
+    margin: 20px 5px;
+  }
+
+%div
+  .alert
+    Hi! This is a skeleton page to see your projects. Visible to admins only ;)
+
+  %div
+    %span New project:
+    .btn-group.level-buttons{role: "group"}
+      %a.btn{href: ''}
+        (TODO) Standalone
+        %i.fa.fa-pencil
+      %a.btn{href: ''}
+        (TODO) Standalone
+        %i.fa.fa-rocket
+      %a.btn{href: '/s/course3/stage/21/puzzle/15'}
```

Comment:
could be.. I was assuming that those paths would go away (or be renamed) because they would be replaced with the new standalone levels